src/ir/auth_logic/BUILD

```diff
@@ -79,13 +79,20 @@ cc_test(
     ],
 )
 
+cc_library(
```

Comment:
why this change? 

[nit: not super particular as the other chagnes are small, but this could be a separate PR.]

Reply:
This is a part of this PR because if I don't make this change, but DO add the universe relation pass as a dependency for the policy_emitter, it causes the build to fail because the implementation of the function is linked twice.

OK this could be a separate PR. Based on your parenthetical, I'll assume you don't insist I split it, but if you do actually insist, say so.